app/models/user.rb

```diff
@@ -12,14 +12,10 @@ class User < ActiveRecord::Base
   validates :last_name, presence: true
   validates :username, presence: true
 
-  enum role: %w(default admin)
-
-  def admin?
-    role == "admin"
-  end
+  enum role: %w(default admin super_admin)
 
   def self.find_or_create_by_auth(auth_data)
-    user = User.where(provider: auth_data['provider'], id: auth_data['uid']).first_or_create
+    user = User.where(provider: auth_data["provider"], id: auth_data['uid']).first_or_create
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.